go/vumitools/tests/test_billing_worker.py

```diff
@@ -45,7 +46,8 @@ def create_transaction(self, account_number, message_id, tag_pool_name,
             "credit_factor": decimal.Decimal('0.4'),
             "created": "2013-10-30T10:42:51.144745+02:00",
             "last_modified": "2013-10-30T10:42:51.144745+02:00",
-            "status": "Completed"
+            "status": "Completed",
+            "transaction_type": BillingDispatcher.TRANSACTION_TYPE_MESSAGE
```

Comment:
Shouldn't this be the ``transaction_type`` that was passed in to ``create_transaction``?